app/controllers/nonprofits/refunds_controller.rb

```diff
@@ -12,8 +12,7 @@ class RefundsController < ApplicationController
     # post /charges/:charge_id/refunds
     def create
```

Comment:
Goodness, this is not good code to start with. It was inherited sadly! Could you try this method with:

```suggestion
    def create
      charge =  current_nonprofit.charges.find(params[:charge_id])
      render_json { InsertRefunds.with_stripe(charge, charge_params.merge(user_id: current_user.id) }
    end
```

Reply:
If line 15 doesn't work with the other suggested change on line 27, please try it with 

`charge_params[:refunds].merge(user_id: current_user.id)`

My mind is struggling to remember which was it's supposed to be :smile: